protocol/tx/spend.go

```diff
@@ -9,17 +9,43 @@ type spend struct {
 		ExtHash     bc.Hash
 	}
 	ordinal int
+
+	// SpentOutput contains (a pointer to) the manifested entry
+	// corresponding to body.SpentOutput. It may be nil, in which case
+	// spentInfo must be non-nil.
+	SpentOutput *output
+
+	// spentInfo contains validation-essential data in case the spent
+	// output object is not present (in SpentOutput).
```

Comment:
Oh it's not just refdata we'd have to add to the db. To fully reconstruct a correctly hashing prevout we also need the prevout's source refhash and "position" (i.e., the pointer into the previous transaction's mux).

This is certainly doable. An interim commit on the `entries` branch did exactly this. But is it desirable?